src/ResourceManager/PowerBIDedicated/Commands.PowerBIDedicated/packages.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Bcl" version="1.1.10" targetFramework="net452" />
```

Comment:
@selasagi you will need to add your management SDK to this file, and you can go ahead and remove all of the other references found in this file.

Reply:
done